Dockerfile.clr.envoy

```diff
@@ -1,63 +1,12 @@
 FROM clearlinux:base as builder
 
-ADD qat1.7.l.4.3.0-00033.tar.gz .
```

Comment:
This seems to be a functional change (pushing the QAT version up for clear linux image). The container built fine though.